CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## [Unreleased]
 
+## [0.1.5] - 2024-??-?? (Pastissos voladors de colors)
```

Comment:
???

Reply:
we're not going to publish, I was only preparing the changelog for when we publish